apps/web/next.config.mjs

```diff
@@ -77,4 +78,42 @@ const config = {
   },
 };
 
-export default bundleAnalyzer(config);
+export default withSentryConfig(
+  bundleAnalyzer(config),
+  {
+    // For all available options, see:
+    // https://github.com/getsentry/sentry-webpack-plugin#options
+
+    // Suppresses source map uploading logs during build
+    silent: true,
+    org: "blossom-labs",
+    project: "blobscan-web",
```

Comment:
These should be configurable. According to the [documentation](https://docs.sentry.io/product/cli/configuration/#configuration-values) we must use SENTRY_ORG and SENTRY_PROJECT